xbmc/guilib/GUIIncludes.h

```diff
@@ -52,7 +52,17 @@ class CGUIIncludes
   const INFO::CSkinVariableString* CreateSkinVariable(const std::string& name, int context);
 
 private:
+  enum RESOLVE_PARAMS_RESULT
```

Comment:
I myself also prefer camel case for enums, but it seems that both styles are used in Kodi. Not sure which one is "right", though.